invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/administration/records/api/routes.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2023 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import _get from "lodash/get";
+
+const APIRoutesGenerators = {
+  delete: (record, idKeyPath = "id") => {
+    return `/api/records/${_get(record, idKeyPath)}/delete`;
```

Comment:
I am ok with this, but I remember that you normally advocate for using record's links instead.